x-pack/plugins/security_solution/common/detection_engine/schemas/common/installed_integrations.ts

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// -------------------------------------------------------------------------------------------------
+// Installed package
+
+/**
+ * Basic information about an installed Fleet package.
+ */
+export interface InstalledPackageBasicInfo {
```

Comment:
We may want to add `iconPath` in the future so we can easily reference the package/integrations icon in the UI. There's no requirement for this now, and would be easy to add later with how you've structured things over in `createInstalledIntegrationSet`, but just wanted to mention this additional data that may be useful.